src/helm/benchmark/presentation/summarize.py

```diff
@@ -738,15 +756,20 @@ def _adapter_spec_sort_key(spec):
 
         if add_win_rate:
             # add overall win rate as the second column
-            win_rates = compute_average_row_win_rates(table)
-            AVERAGE_WIN_RATE_COLUMN = 1
+            WIN_RATE_AGGREGATION = "mean"
+            win_rates = compute_aggregate_row_win_rates(table, aggregation=WIN_RATE_AGGREGATION)
+            AGGREGATE_WIN_RATE_COLUMN = 1
             description = "How many models this model outperform on average (over columns)."
             table.header.insert(
-                AVERAGE_WIN_RATE_COLUMN,
-                HeaderCell(f"Average win rate {UP_ARROW}", description=description, lower_is_better=False),
+                AGGREGATE_WIN_RATE_COLUMN,
+                HeaderCell(
+                    f"{WIN_RATE_AGGREGATION.capitalize()} win rate {UP_ARROW}",
```

Comment:
If we already have `lower_is_better`, then we don't need to put the `UP_ARROW` in the backend logic, but it should be rendered on the frontend (basically, backend shouldn't know about arrows).

Reply:
Makes sense, moved all arrow logic to the frontend, now we only have `lower_is_better` in the backend.

@yifanmai made a small change so that `sortOrder` uses lower_is_better from the table file directly. It looks fine in my testing but just wanted to cc you.